src/style.css

```diff
@@ -0,0 +1,101 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300, 900;
+}
+
+body {
+  margin: 0;
+}
+
+.data-qa {
```

Comment:
data-qa it's attribute for tests, for your styling better use other names 'searchInput' for example